rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -226,15 +231,15 @@ def getNumber(self, index, fieldName):
 			ctypes.byref(c_value),
 			tocstring(self.input.name),
 			index,
-			tocstring(fieldName))
+			tocstring(field_name))
 		_check_retcode(retcode)
 
 		if retcode == _ReturnCode.no_data:
 			return None
 
 		return c_value.value
 
-	def getBoolean(self, index, fieldName):
+	def getBoolean(self, index, field_name):
```

Comment:
How are we going about naming of methods in Python now? It seems that we have some using camel_case and others using snakeCase (and some both)

Reply:
We're doing `snake_case` for methods and variables, and CamelCase for types, which is the recommended Python style